addons/mail/static/tests/owl/component/composer_tests.js

```diff
@@ -175,17 +175,23 @@ QUnit.test('add emoji replaces (keyboard) text selection', async function (asser
     await nextAnimationFrame();
 });
 
-// Test skipped until mentions manager is ready
-QUnit.skip('display partner mention suggestions on typing "@"', async function (assert) {
+QUnit.test('display partner mention suggestions on typing "@"', async function (assert) {
     assert.expect(2);
 
-    await this.start();
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.method === 'get_mention_suggestions') {
+
+            }
+            return this._super(...arguments);
+        }
+    });
     const composerLocalId = this.env.store.dispatch('_createComposer');
     await this.createComposerComponent(composerLocalId);
     assert.strictEqual(
-        document.querySelectorAll(`.tribute-container`).length,
-        0,
-        "should not display the tribute mention suggestions initially"
+        document.querySelector('.o_ComposerTextInput_mention_dropdown .dropdown-menu').classList.contains('show'),
```

Comment:
looks like it should be assert.containsNone and I find the selector a bit too complex or specific here

Reply:
Done